utils/api/benefitHandler.ts

```diff
@@ -210,7 +211,11 @@ export class BenefitHandler {
         if (this.input.client.income.partnerAvailable)
           requiredFields.push(FieldKey.PARTNER_INCOME)
       }
-      if (this.input.client.partnerBenefitStatus.helpMe) {
+      if (
+        this.input.client.partnerBenefitStatus.helpMe &&
+        this.input.partner.age >= 60 &&
+        this.input.partner.age <= 64
```

Comment:
somehow this is not working for me, I entered 1958 = 64 and it wont ask the question i also entered 1963 = 59 and the questions were asked ?